### PR TITLE
Get rid of espec

### PR DESCRIPTION
**Pull request by varsill**
This PR removes the ESpec dependency, removes unecessary ESpec tests and rewrites the last two significant ESpec tests (input queue test and caps matcher test) with ExUnit.

### CODE REVIEW

File: test/membrane/core/element/input_queue_test.exs [hunk 1]
```diff
@@ -1,9 +1,187 @@
...
+             }
+
+      message =
+        Message.new(:demand, context.target_queue_size, for_pad: context.linked_output_ref)
+
+      assert_received ^message
+    end
+
+    test "send the demand if toilet is enabled not", context do
+      flush()
```
Comment (mat-hek) [inline test/membrane/core/element/input_queue_test.exs:53]: Each exunit test is run in a separate process, so I think it's not needed ;)

File: mix.exs [hunk 2]
```diff
@@ -16,9 +16,8 @@ defmodule Membrane.Mixfile do
       name: "Membrane Core",
       source_url: link(),
       docs: docs(),
-      aliases: ["test.all": "do espec, test", docs: ["docs", &copy_assets/1]],
```
Comment (bblaszkow06) [inline mix.exs:19]: I'd remove "test.all" altogether.

**Review by mat-hek — APPROVED** [review]

**Comment by mat-hek** [comment]
Let's merge this after all the playback stuff is merged

File: mix.exs [hunk 3]
```diff
@@ -16,16 +16,15 @@ defmodule Membrane.Mixfile do
...
+      aliases: [docs: ["docs", &copy_assets/1]],
       preferred_cli_env: [
-        espec: :test,
-        "test.all": :test,
+        test: :test,
```
Comment (bblaszkow06) [inline mix.exs:21]: This isn't needed, `test` task already specifies test as its preferred env (this option allows to override it).
```suggestion
```

**Review by bblaszkow06 — APPROVED** [review]

File: mix.exs (hunk 3 above)
Comment (varsill) [reply mix.exs:21]: done

**Pull request merged**